src/App.vue

```diff
@@ -21,14 +21,38 @@ import FullWidth from '@/components/layouts/FullWidth.vue'
 import Navbar from '@/components/Navbar.vue'
 import MobileNavbar from '@/components/MobileNavbar.vue'
 
-import { between, required, email } from 'vee-validate/dist/rules'
+import { between, required, email, dimensions, image } from 'vee-validate/dist/rules'
 import { extend } from 'vee-validate'
 
+var validateImageDimensions = function (value, { width, height }) {
+  return new Promise(function (resolve) {
+    var image = new Image()
+    image.onerror = function () { return resolve(false) }
+    image.onload = function () { return resolve(image.width >= width && image.height >= height) }
+    image.src = URL.createObjectURL(value[0])
+  })
+}
+
+var validateImage = function (files) {
+  var regex = /\.(heic|heif|tiff|webp|ico|jpg|svg|jpeg|png|bmp|gif)$/i
+  if (Array.isArray(files)) {
+    return files.every(function (file) { return regex.test(file.name) })
+  }
+  return regex.test(files.name)
+}
+
```

Comment:
Maybe const or let.. ? It is a good practise to avoid anonymous functions, due to better readability. What you can do is either name a function or use ES6 arrow function.

Reply:
I addressed this issue in 5a7c4e22a3ca45327b68a7a73b93c1b1403b927e. I think that this is what you mean that i am copying code. I did this because vee-validate checks for equal dimensions of the image, and i modified it to check whether image has greater width and height than defined in validation rules. Vee validate also lacks option to define what extensions image can have, so i declared a list of extensions user may use.